.gitignore

```diff
@@ -7,8 +7,6 @@
 /.env
 /.DS_Store
 __pycache__/
-/authentication/migrations/
-/profiles/migrations/
 /node_modules
 package-lock.json
 package.json
```

Comment:
Must be included.

Reply:
> Must be included.

Do I understand correctly? They are necessary for the evolution of the database schema, but there is no need to include them in the git because developers can generate them with the makemigrations command?